discovery-provider/alembic/versions/58b26285cdcf_migrate_triggers_for_notifiations.py

```diff
@@ -0,0 +1,54 @@
+"""migrate triggers for notifiations
+
+Revision ID: 58b26285cdcf
+Revises: b0623220e904
+Create Date: 2022-07-13 16:02:12.264201
+
+"""
+from pathlib import Path
+
+import sqlalchemy
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "58b26285cdcf"
+down_revision = "b0623220e904"
+branch_labels = None
+depends_on = None
+
+
+def load_sql(name):
```

Comment:
if you want you can use helper function a-la:
https://github.com/AudiusProject/audius-protocol/pull/3623/files#diff-0cdc2dd9e07a47f94e4539504391d2e4bb5e6efe39c6a9b9b0e79ceadc612ec7

But this is also fine.